jest.test.config.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     // - https://github.com/kulshekhar/ts-jest/blob/master/docs/user/config/isolatedModules.md
     'ts-jest': {
       isolatedModules: true,
+      babelConfig: true,
```

Comment:
Otherwise the babel plugins won't run at all. Which means that e.g. the emotion plugin doesn't run which is needed for component selectors.

https://kulshekhar.github.io/ts-jest/docs/getting-started/options/babelConfig/